build-support/bin/BUILD

```diff
@@ -16,13 +16,17 @@ python_sources(
 
 python_tests(name="py_tests", overrides={"reversion_test.py": {"timeout": 90}})
 
-pex_binary(name="changelog", entry_point="changelog.py")
-pex_binary(name="check_banned_imports", entry_point="check_banned_imports.py")
-pex_binary(name="check_inits", entry_point="check_inits.py")
-pex_binary(name="deploy_to_s3", entry_point="deploy_to_s3.py")
-pex_binary(name="generate_all_lockfiles_helper", entry_point="_generate_all_lockfiles_helper.py")
-pex_binary(name="generate_docs", entry_point="generate_docs.py")
-pex_binary(name="generate_github_workflows", entry_point="generate_github_workflows.py")
-pex_binary(name="generate_user_list", entry_point="generate_user_list.py")
-pex_binary(name="release_helper", entry_point="_release_helper.py")
-pex_binary(name="reversion", entry_point="reversion.py")
+pex_binaries_from_entry_points(
```

Comment:
Given that the field is called `entry_points` instead of `sources`, I think this target name is overly redundant now. How about simply calling it `pex_binaries`?

I do still think it's a bad idea to use this target generator when the entry-point isn't a first-party source. But calling it `pex_binaries_from_sources` is confusing because I agree with you that it should be `entry_points` rather than `sources`.

Reply:
I'd love that :kissing_heart: